eagle-assembly/src/main/README.md

```diff
@@ -1,3 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Thanks for catching that. I was not familiar with Markdown syntax. will update the patch shortly.